diff --git a/dist/chunks/test.DqQZzsWf.js b/dist/chunks/test.DqQZzsWf.js
index 1992f7806544a527ebdd12f876d8dcdfa25da793..657cc0c18916fb4dcd9bcd9e69c7535dd9eb5337 100644
--- a/dist/chunks/test.DqQZzsWf.js
+++ b/dist/chunks/test.DqQZzsWf.js
@@ -41,14 +41,17 @@ async function runBenchmarkSuite(suite, runner) {
 		updateTask$1("suite-prepare", suite);
 		const addBenchTaskListener = (task, benchmark) => {
 			task.addEventListener("complete", (e) => {
-				const taskRes = e.task.result;
+				const taskRes = e.task.result.latency;
 				const result = benchmark.result.benchmark;
 				benchmark.result.state = "pass";
 				Object.assign(result, taskRes);
 				// compute extra stats and free raw samples as early as possible
+				result.samples ??= [];
 				const samples = result.samples;
-				result.sampleCount = samples.length;
-				result.median = samples.length % 2 ? samples[Math.floor(samples.length / 2)] : (samples[samples.length / 2] + samples[samples.length / 2 - 1]) / 2;
+				const sampleCount = result.samplesCount;
+				result.sampleCount = sampleCount;
+				result.median = result.p50;
+				result.hz = e.task.result.throughput.mean;
 				if (!runner.config.benchmark?.includeSamples) result.samples.length = 0;
 				updateTask$1("test-finished", benchmark);
 			}, { once: true });
